css/style.css

```diff
@@ -60,6 +60,47 @@ form li {
   transition: transform 0.2s;
 }
 
+.m-menu-continer {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 700px;
+  width: 100%;
+  background-color: rgb(96, 112, 255, 0.9);
+}
+
+.close {
+  width: 12px;
+  height: 12px;
+  margin-left: 90%;
+  margin-top: 24px;
+  margin-right: 34px;
+  display: block;
+  color: #fff;
+  opacity: 100%;
+}
+
+.links-container {
+  list-style: none;
+  padding: 24px 24px;
+}
+
+.links {
+  color: #fff;
+  text-decoration: none;
+  font-size: 32px;
+}
+
+.m-menu-list {
+  margin-bottom: 30px;
+  padding-left: 20px;
+}
+
+.m-menu-continer.active {
+  display: block;
+}
+
 .home {

```

Comment:
- [ ] Great job so far! 👏🏻  however according to the project requirement, 🤔  on the desktop size, the hamburger menu button, the closed button, and the menu mobile page must not be visible. Kindly correct this. You can use your `@media screen and (min-width: 768px) {`  to display none of your navigation menu. 😃 

- Your design:

![20bad](https://user-images.githubusercontent.com/79058364/174663724-4b5a7bfd-1220-4dd1-b433-8b4e0bbdcc86.png)